registry.k8s.io/images/k8s-staging-cluster-api-vsphere/OWNERS

```diff
@@ -0,0 +1,19 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- chrischdi
+- gab-satchi
+- killianmuldoon
+- randomvariable
+- sbueringer
+- srm09
+- yastij
+
+emeritus_approvers:
```

Comment:
Any reason why this list is the same as the approvers ?

Reply:
Copy & pasted and thought one of those sections is reviewers. Thx!